builtin/providers/openstack/resource_openstack_compute_instance_v2.go

```diff
@@ -998,7 +998,7 @@ func getInstanceAccessAddresses(d *schema.ResourceData, networks []map[string]in
 			hostv6 = n["fixed_ip_v6"].(string)
 		}
 
-		if n["access_network"].(bool) {
+		if ok, an := n["access_network"].(bool); ok && an {
```

Comment:
Should these be named the opposite way around for clarity? Appreciate that it doesn't actually matter here, but the "ok" is generally the second parameter of the type assertion?